ClosedXML/Excel/CalcEngine/Expression.cs

```diff
@@ -354,12 +354,21 @@ override public object Evaluate()
                     return (double)_lft * (double)_rgt;
 
                 case TKID.DIV:
+                    if (Math.Abs((double)_rgt) < double.Epsilon)
```

Comment:
Perhaps we should introduce `XLHelper.Epsilon` of approximately `2.2250738585073E-308`, because `double.Epsilon` is 16 decimal orders lower (4.9somethingE **-324**) then Excel supports.

Reply:
So you're saying the ClosedXML calculation engine has a higher precision than Excel? If that's true, I wouldn't want to change it. It's a useful feature, not a bug.

Reply:
Let it be so